docs/man/ntp.toml.5.md

```diff
@@ -14,61 +14,34 @@ allowing a json-like syntax.
 
 The ntpd-rs configuration file consists of several sections, each of which
 configuring a separate part of the ntp-daemon process. Each of the secions is
-described in the rest of this document. Many settings will have defaults,
-which will be indicated by each configuration setting shown.
+described in the rest of this document. Many settings will have defaults, which
+will be indicated by each configuration setting shown.
 
 For those familiar with the NTP protocol: ntpd-rs only supports unicast
 client-server connections and the concepts in ntpd-rs are all built up around
-that concept. Most NTP traffic, especially across the public internet, almost
+that idea. Most NTP traffic, especially across the public internet, almost
```

Comment:
The first sentence of this paragraph feels weird, not really sure what we are trying to say here.